pywavefront/obj.py

```diff
@@ -29,15 +33,42 @@ def __init__(self, wavefront, file_name, strict=False, encoding="utf-8", create_
         self.mesh = None
         self.material = None
         self.create_materials = create_materials
+        self.meta = Meta()
+        self.cache = cache
+        self.cache_loaded = False
+        self.parse_now = parse
```

Comment:
Can you explain what this does? I see one reference to `self.parse_now` further down in `__init__`, but nowhere else.